e2e-tests/portal/cypress/e2e/journeys/dashboard/dashboard-deals/filter-by/dashboard-deals-filter-by-multiple-fields-multiple-values.spec.js

```diff
@@ -158,10 +160,14 @@ context('Dashboard Deals filters - filter by multiple fields with multiple value
   });
 
```

Comment:
This test and others are still somehow using `cy.insertOneDeal`, there should be no instances of this and the command should be removed.